### PR TITLE
Add list of hosts to yandex_mdb_kafka_cluster attributes

### PR DESCRIPTION
**Pull request by alxrem**

### CODE REVIEW

**Comment by alxrem** [comment]
@apilikov Should I do something more for this PR to be accepted?

**Comment by apilikov** [comment]
Please add CHANGELOG record. Make sure it conforms to the style.

**Comment by alxrem** [comment]
@apilikov Done

File: website/docs/d/datasource_mdb_kafka_cluster.html.markdown [hunk 1]
```diff
@@ -117,3 +118,12 @@ The `topic_config` block supports:
 `retention_bytes`, `retention_ms`, `max_message_bytes`, `min_insync_replicas`, `segment_bytes`, `preallocate`, - (Optional) Kafka topic settings. For more information, see
 [the official documentation](https://cloud.yandex.ru/docs/managed-kafka/operations/cluster-topics#update-topic)
 and [the Kafka documentation](https://kafka.apache.org/documentation/#configuration).
+
+The `host` block supports:
+
+* `name` - The fully qualified domain name of the host.
+* `zone` - The availability zone where the Kafka host was created.
```
Comment (vbelov) [inline website/docs/d/datasource_mdb_kafka_cluster.html.markdown]:
```suggestion
* `zone_id` - The availability zone where the Kafka host was created.
```

File: website/docs/r/mdb_kafka_cluster.html.markdown [hunk 2]
```diff
@@ -358,6 +358,17 @@ In addition to the arguments listed above, the following computed attributes are
 * `status` - Status of the cluster. Can be either `CREATING`, `STARTING`, `RUNNING`, `UPDATING`, `STOPPING`, `STOPPED`, `ERROR` or `STATUS_UNKNOWN`.
   For more information see `status` field of JSON representation in [the official documentation](https://cloud.yandex.com/docs/managed-kafka/api-ref/Cluster/).
 
+* `host` - A host of the Kafka cluster. The structure is documented below.
+
+The `host` block supports:
+
+* `name` - The fully qualified domain name of the host.
+* `zone` - The availability zone where the Kafka host was created.
```
Comment (vbelov) [inline website/docs/r/mdb_kafka_cluster.html.markdown]:
```suggestion
* `zone_id` - The availability zone where the Kafka host was created.
```

**Review by vbelov — CHANGES_REQUESTED** [review]

**Comment by apilikov** [comment]
Please resolve conflicts.

**Review by apilikov — APPROVED** [review]

**Comment by apilikov** [comment]
Thank you!

**Pull request merged**